tests/integration/commands/init_command_test.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+from filecmp import dircmp, cmpfiles
+
+import pytest
+
+from protostar.cairo import CairoVersion
+from tests.integration.conftest import CreateProtostarProjectFixture
+from tests.conftest import PROJECT_ROOT
+
+
+@pytest.mark.parametrize(
+    "create_protostar_project, cairo_version",
+    [
+        (CairoVersion.cairo0, CairoVersion.cairo0),
+        (CairoVersion.cairo1, CairoVersion.cairo1),
+    ],
+    indirect=["create_protostar_project"],
+)
+def test_init(
+    create_protostar_project: CreateProtostarProjectFixture, cairo_version: CairoVersion
+):
```

Comment:
Read [this comment](https://github.com/software-mansion/protostar/pull/1564/files#r1126353789) first.

There we pass "create_protostar_project" as [indirect](https://docs.pytest.org/en/latest/example/parametrize.html#apply-indirect-on-particular-arguments) argument to let pytest know it has pass the first argument of the tuple (the second one cairo_version) to the request object that is passed to the fixture [there](https://github.com/software-mansion/protostar/pull/1564/files#diff-c87c5173e88bb0f60b00107f4586304f4869a976a4f998425efe3481c0db9f56R142-R147)